README.md

```diff
@@ -1,58 +1,91 @@
+- __Onboarding and knowledge transfer:__ Reviews help us onboard new team members by exposing them to our codebases and practices.
 
 ## PR Expectations
 
-### PR Description
-
-- PR naming conventions
-- Title
-- Link to ticket
-- What's Here: Describe what you did and why, before and after
-- Steps to test
-- Questions/What I'd Like Feedback On
-- Other References
-
 ### Code Expectations
 
-- PRs should be small both so that reviews are digestible and to reduce deployment risks
-- Code should conform to linting/style expectations
-- All functions/methods should have comment blocks in the standard documentation format for the language (generally docblock)
-- Unit test coverage is expected and reported
-- Commits follow semantic commits
+- PRs should be small. Meaningful reviews require digestible changes. Small changes also reduce deployment risks.
+- Code should conform to linting/style expectations.
+- All functions/methods should have comment blocks in the standard documentation format for the language (generally docblock).
+- Ideally, all code should be checked in alongside tests that validate its
+  correctness. These should include unit tests, at a minimum. Larger features
+  should include integration tests (possibly in parallel PRs, depending on
+  feature size).
+- Use semantic commit messages.
+
+### PR Format
+
+PRs should include:
+
+- __A meaningful and descriptive title, chosen according to naming conventions.__ (TBD)
+- __Links to any relevant tickets or issues.__
+- __A description of "What's Here."__ Describe what you did and _why_. This
+  description should include the previous behavior of the code and how the new
+  behavior differs.
+- __Steps to test.__ Reviewers should be able to run the code themselves in
+  order to explore and validate its behavior.
+- __Questions/What you'd like feedback on.__ Call out any specific areas of the
+  code or concepts that you would like people to focus on.
+- __References.__ Include links to design docs/PRDs, runs of tests/benchmarks,
+  etc.
 
 ### Review Etiquette and Expectations
 
-- Spend the time to read, understand, and preferably test what's in the PR -- time spent here benefits all of us and our users.
-- Ask questions - don't assume that you know why the author made the choices they did.
-- Try to understand the big picture -- where this code is going, how it's meant to serve users and how it will fit into our codebase.
-- Use the PR forum for contructive discussion
-- Always consider perfomance, security, and adherence to existing mental models and patterns.
-- An approval means you're confident that this PR will both do what it's supposed to do, improve the Mozilla Social codebase, and not introduce technical debt.
-- Expect to complete your review within 1 business day of the PR going to review. If you can't do that, please tell the author early on.
-
-### Reviewer Expectations
+- __First and foremost: use the PR forum for _constructive_ discussion.__ Discuss
```

Comment:
Should we add something about following the [Mozilla Participation Guidelines ](https://www.mozilla.org/en-US/about/governance/policies/participation/) and to note that there are always multiple ways to code things.

Reply:
100% yes, I added something at the top -- PTAL